tests/idris2/perror008/expected

```diff
@@ -0,0 +1,8 @@
+1/1: Building MultilineError (MultilineError.idr)
```

Comment:
This duplicates https://github.com/idris-lang/Idris2/blob/1d692dae6b418e00ed30d28b904c463e029d61d7/tests/idris2/perror007/expected#L82-L88